extension/healthcheckextension/README.md

```diff
@@ -33,6 +32,12 @@ extensions:
   health_check:
   health_check/1:
     endpoint: "localhost:13"
+    tls:
+      ca_file: "/path/to/ca.crt"
+      cert_file: "/path/to/cert.crt"
+      key_file: "/path/to/key.key"
+    auth:
+      authenticator: "basicauth"
```

Comment:
The authenticator might be confusing here for readers, as you are not specifying it anywhere.

Reply:
Noted. I have removed this entry.